parsers/ansibleplaybook.c

```diff
@@ -185,10 +185,11 @@ static void ansiblePlaybook (yaml_token_t *token, void *data)
 
 static void *ansiblePlaybookPrepare(void)
 {
-	static struct ansiblePlaybookState state = {
-		.type_stack = NULL,
-		.play_detection_state = DSTAT_PLAY_NAME_INITIAL,
-	};
+	static struct ansiblePlaybookState state;
+
+	Assert (state.type_stack = NULL);
```

Comment:
is it really a good idea to perform such a check in the init phase?  If it's useful, shouldn't it rather be at the end of the previous run, as IIUC it checks whether the state was properly poped or something?